postgres/tests/test_pg_integration.py

```diff
@@ -217,6 +233,39 @@ def test_config_tags_is_unchanged_between_checks(integration_check, pg_instance)
         assert check.config.tags == expected_tags
 
 
+def test_statement_metrics(aggregator, integration_check, pg_instance):
+    pg_instance['deep_database_monitoring'] = True
+    # The query signature should match the query and consistency of this tag has product impact. Do not change
+    # the query signature for this test unless you know what you're doing.
+    QUERY = 'select * from pg_stat_activity'
+    QUERY_SIGNATURE = '7cde606dbf8eaa17'
+
+    check = integration_check(pg_instance)
+    check._connect()
+    cursor = check.db.cursor()
+
+    # Execute the query once to begin tracking it. Execute again between checks to track the difference.
+    # This should result in a count of 1
+    cursor.execute(QUERY)
+    check.check(pg_instance)
+    cursor.execute(QUERY)
```

Comment:
Why are these query needed?

Reply:
The first query creates a row in the `pg_stat_statements` table for that query statement. At this point, when the check runs, it sees the query for the first time and stores it in cache.

The second query increments the counters on `pg_stat_statements`. The next check run picks up those differences in that query. It should then submit `postgresql.queries.*` metrics for that query. 

I.e. the tags should have the tag `'query:{}'.format(QUERY)` as well as the proper `query_signature` tag.